fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -2126,6 +2129,61 @@ private void handleCtasRollback(TableName table) {
         }
     }
 
+    private void handleIotsStmt() {
+        InsertOverwriteTableSelect iotsStmt = (InsertOverwriteTableSelect) this.parsedStmt;
+        try {
+            // create table
+            DdlExecutor.execute(context.getEnv(), iotsStmt.getCreateTableLikeStmt());
+            context.getState().setOk();
+        } catch (Exception e) {
+            // Maybe our bug
+            LOG.warn("IOTS create table error, stmt={}", originStmt.originStmt, e);
+            context.getState().setError(ErrorCode.ERR_UNKNOWN_ERROR, "Unexpected exception: " + e.getMessage());
```

Comment:
just return if error happens.
And no need to check the `context.getState().getStateType()` below.